FreeRTOS_Sockets.c

```diff
@@ -3067,13 +3052,6 @@ void vSocketWakeUpUser( FreeRTOS_Socket_t * pxSocket )
                 /* Did it get connected while sleeping ? */
                 xResult = FreeRTOS_issocketconnected( pxSocket );
 
-                /* Returns positive when connected, negative means an error */
-                if( xResult < 0 )
-                {
-                    /* Return the error */
-                    break;
-                }
```

Comment:
The socket has already been checked, so `FreeRTOS_issocketconnected()` won't return an error.